string-manipulation-1/src/is-upper-cased.js

```diff
@@ -0,0 +1,9 @@
+/* exported isUpperCased */
+function isUpperCased(word) {
+  var wordUpperCased = word.toUpperCase();
+  if (word === wordUpperCased) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
Same note as above ^. You can also handle this is one line if you wanted to
`return word === word.toUpperCase()`